tracer/src/Datadog.Trace/Util/RuntimeId.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="RuntimeId.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Util
+{
+    internal static class RuntimeId
+    {
+#if NETFRAMEWORK
+        private const string NativeLoaderLibNameX86 = "Datadog.AutoInstrumentation.NativeLoader.x86.dll";
+        private const string NativeLoaderLibNameX64 = "Datadog.AutoInstrumentation.NativeLoader.x64.dll";
+#else
+        private const string NativeLoaderLibNameX86 = "Datadog.AutoInstrumentation.NativeLoader.x86";
+        private const string NativeLoaderLibNameX64 = "Datadog.AutoInstrumentation.NativeLoader.x64";
```

Comment:
I think there's a non-zero change of .NET Framework loading the dll that targets .NET Standard. Will this still work in that case? In other places we determine the runtime at run time (no pun intended) instead of doing it at compile time.

Reply:
Without the extension, it will [still work](https://docs.microsoft.com/en-us/dotnet/standard/native-interop/cross-platform) 
To be honest, I do not remember why we added the extension. I know at some point I tried without but I had an issue.

> In other places we determine the runtime at run time (no pun intended) instead of doing it at compile time.

Yes, @tonyredondo suggested it to avoid runtime issue, but I did not see the issue we would have (except dll being renamed) and he did not recall the actual case.
We agreed on leaving it like that for now. But if you remember something, I would like to know (and I'll fix it right away :) )